audit_exceptions/versioned_head_spec_allowlist.json

```diff
@@ -2,5 +2,10 @@
   "bash-completion@2",
   "botan@2",
   "imagemagick@6",
-  "mbedtls@2"
+  "mbedtls@2",
+  "postgresql@12",
+  "postgresql@13",
+  "postgresql@14",
+  "postgresql@15",
+  "postgresql@16"
```

Comment:
I didn't spot this before, but adding these to an allowlist is a pretty clear indication this is not normally a supported configuration

Reply:
I had inquired about this in Homebrew/brew#15218 and I was told to do it this way.